frontend/components/queries/QueryPageWrapper/helpers.js

```diff
@@ -7,9 +7,10 @@ import { renderFlash } from "redux/nodes/notifications/actions";
 
 export const fetchQuery = (dispatch, queryID) => {
   return dispatch(queryActions.load(queryID)).catch((errors) => {
-    const errorMessage = join(values(omit(errors, "http_status")), ", ");
+    const { MANAGE_QUERIES } = PATHS;
+    const errorMessage = "The query you requested does not exist in Fleet.";
```

Comment:
Can we add an `if` here that checks for the error text `no rows in result set` (which should always be returned if the error is due to no matching query)? If the error doesn't match that, it could be helpful to render the actual error so that we don't mislead the user.